src/avfoundation.cs

```diff
@@ -13140,7 +13375,7 @@ interface AVContentKeySession {
 	}
 
 	[Static][Internal]
-	[NoWatch, NoTV, Mac (10,15), iOS (12,2)]
+	[Watch (2,0), NoTV, Mac (10,15), iOS (12,2)]
```

Comment:
hmm... I don't think AVFoundation was available in 2.0

https://developer.apple.com/documentation/avfoundation?language=objc suggest it was in 3.0
In about better use the current SDK as it was likely not very useful before now